web/web/config.py

```diff
@@ -31,3 +32,7 @@ class DevelopmentConfig(Config):
     SQLALCHEMY_ECHO = False
     DB_USER = "tess_user"
     DB_PASSWORD = "tess_db_password_local"
+    JWT_ACCESS_TOKEN_EXPIRES = timedelta(minutes=15)
```

Comment:
let's make this longer 😬, 15 minutes is too short especially during development.

Reply:
How many minutes would you recommend?

Reply:
The two hours you added is fine. My guess is that once people actually start using this we'll likely go to something like 15-30 days. Especially if we implement a "keep me logged"/"remember me" feature.